test/test_helpers/chdir.cpp

```diff
@@ -5,15 +5,15 @@
 
 #include "utils.h"
 
-test_helpers::Chdir::Chdir(const std::string& path)
+test_helpers::Chdir::Chdir(const newsboat::Filepath& path)
 {
 	m_old_path = newsboat::utils::getcwd();
-	const int result = ::chdir(path.c_str());
+	const int result = ::chdir(std::string(path).c_str());
 	if (result != 0) {
 		const auto saved_errno = errno;
 		auto msg = std::string("test_helpers::Chdir: ")
 			+ "couldn't change current directory to `"
-			+ path
+			+ std::string(path)
```

Comment:
This message is going to be displayed to the user, so we should use `.display()` here to get a valid UTF-8 string.